.github/workflows/build.yaml

```diff
@@ -28,6 +28,9 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       scripts: ${{ steps.package.outputs.scripts }}
+      repoNames: ${{ steps.package.outputs.repoNames }}
+      repoFolders: ${{ steps.package.outputs.repoFolders }}
+      systemJSFiles: ${{ steps.package.outputs.systemJSFiles }}
```

Comment:
I don't like this actually. Maybe we only need one array and we can get the other info in a small setup part inside the matrix

Reply:
See my suggested output from 'Get repo matrix' in the scan-packages action